Tests/test_SeqIO_index.py

```diff
@@ -111,6 +112,20 @@ def test_old(self):
             self.assertEqual(54, len(d))
             self.assertRaises(FileNotFoundError, d.get_raw, "alpha")
 
+        def test_old_check_same_thread(self):
+            """Setting check_same_thread to False doesn't raise an exception"""
```

Comment:
Needs a trailing full stop,

```
$ flake8
./Tests/test_SeqIO_index.py:116:1: D400 First line should end with a period
```

Reply:
Strange - you're already in the top level Biopython folder so https://gitlab.com/pycqa/flake8/issues/517 shouldn't be the problem. Maybe the version of flake8 is important...

Reply:
As to the task at hand, we currenlty tell flake8 to ignore the following under ``Tests/``:

F401,F841,D101,D102,D103,B009,B010,B011,C812

That is on top of the things we ignore everywhere like E501 - hopefully the ``.flake8`` file comments are clear.

i.e. D400 is the only one you should be seeing here.